extensions/smallrye-graphql/deployment/src/test/java/io/quarkus/smallrye/graphql/deployment/SecurityTest.java

```diff
@@ -0,0 +1,77 @@
+package io.quarkus.smallrye.graphql.deployment;
+
+import static org.hamcrest.Matchers.*;
```

Comment:
We try to avoid star imports but as it's a test, I'll let it slide.

Reply:
Fixed